libs/core/tests/unit_tests/stores/test_in_memory.py

```diff
@@ -1,6 +1,36 @@
+from typing import Tuple
+
+import pytest
+from langchain_standard_tests.integration_tests import (
+    BaseStoreAsyncTests,
+    BaseStoreSyncTests,
+)
+from langchain_standard_tests.integration_tests.base_store import V
+
 from langchain_core.stores import InMemoryStore
 
 
+# Check against standard tests
+class TestSyncInMemoryStore(BaseStoreSyncTests):
```

Comment:
Added unit tests for in memory store and confirmed that the assertions work well